File: Templates/IBM_Cloud/openwhisk-nodejs/handler.js
'use strict';

const code_logic = require("./code_logic");

function hello(params) {
  const ExecutionStartTime = Date.now();

//  const name = params.name || 'World';

  const result = code_logic();

  const ExecutionEndTime = Date.now();

//  console.log('log', { payload: `Hello, ${name}` });

//  return { payload: `Hello, ${name}!` };

  return {
    statusCode: 200,
//    body: JSON.stringify(
//      {
        message: 'Go Serverless v1.0! Your function executed successfully!',
        input: params,
        res: result,
        executionStart: ExecutionStartTime,
        executionEnd: ExecutionEndTime,
//      },
//      null,
//      2
//    ),
  };



}

exports.hello = hello;

File: Templates/IBM_Cloud/openwhisk-python/handler.py
import json
import sys
import time, datetime


def hello(params):

    
    ExecutionStartTime = int(round(time.time() * 1000))
    
    sys.path.insert(0, '/code_logic')
    
    from code_logic import code_logic
    
    result = code_logic.code()
    
    ExecutionEndTime = int(round(time.time() * 1000))



#    body = {
#        "message": "Go Serverless v1.0! Your function executed successfully!",
#        "input": event,
#        "res": result,
#        "ExecutionStart": ExecutionStartTime,
#        "ExecutionEnd": ExecutionEndTime
#    }

    response = {
        "statusCode": 200,
#        "body": json.dumps(body)
        "message": "Go Serverless v1.0! Your function executed successfully!",
#	"input": params,
	"res": result,
	"executionStart": ExecutionStartTime,
	"executionEnd": ExecutionEndTime
    }

    return response

#    name = params.get("name", "stranger")
#    greeting = "Hello " + name + "!"
#    print(greeting)
#    return {"greeting": greeting}
